tests/libtest/thrift_client_to_db.py

```diff
@@ -194,23 +202,26 @@ def get_all_run_results(client, run_id, sort_mode=[], filters=[]):
     return results
 
 
-def get_viewer_client(port, host='localhost', uri='/',
+def get_viewer_client(product, port, host='localhost',
+                      endpoint='/CodeCheckerService',
                       auto_handle_connection=True,
                       session_token=None):
 
-    return CCViewerHelper(host,
-                          port,
-                          uri,
+    return CCViewerHelper(host, port, product,
+                          endpoint,
                           auto_handle_connection,
                           session_token)
 
 
-def get_server_client(port, host='localhost'):
-    return CCReportHelper(host, port)
-
-
 def get_auth_client(port, host='localhost', uri='/Authentication',
                     auto_handle_connection=True, session_token=None):
     return CCAuthHelper(host, port, uri,
                         auto_handle_connection,
                         session_token)
+
+
+def get_product_client(port, host='localhost', uri='/Products',
```

Comment:
These even more low-level stubs exist here so theoretically, the test uses uses the least possible code from `libcodechecker`.